spec/index.bs

```diff
@@ -6475,8 +6491,24 @@ following members:
 
                                     :  {{GPUBindGroupLayoutEntry/externalTexture}}
                                     ::
-                                        - |resource| is a {{GPUExternalTexture}}.
+                                        - |resource| is either a {{GPUExternalTexture}} or a {{GPUTextureView}}.
                                         - |resource| is [$valid to use with$] |this|.
+                                        - If |resource| is a:
+                                            <dl class=switch>
+                                                : {{GPUTextureView}}
+                                                ::
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/usage}}
+                                                        must include {{GPUTextureUsage/TEXTURE_BINDING}}.
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/dimension}}
+                                                        must be {{GPUTextureViewDimension/"2d"}}.
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/mipLevelCount}}
+                                                        must be 1.
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}
+                                                        must be a [=filterable format=] listed in
+                                                        [[#plain-color-formats]] table.
```

Comment:
I think the only remaining issue was some hesitation over supporting all formats here. I do not think it is really useful to support all formats, so perhaps we should make things simpler by supporting a very limited set like `rgba8unorm` and `rgba16float`. (No srgb, no bgra, no worrying about `"float32-filterable"`.)

Reply:
We should involve all the implementers given that the discussion in the group was about concerns that some implementation workarounds could interact badly with this addition to the API.